test_commanderror_jsonmsgs fails when tested against our minimal GIT_ANNEX_MIN_VERSION = '8.20200309'

that matrix run is a cron-only run. sample log
https://app.travis-ci.com/github/datalad/datalad/jobs/528098029

```
======================================================================
FAIL: datalad.cmdline.tests.test_main.test_commanderror_jsonmsgs
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/tmp/dl-miniconda-0yykfkkk/lib/python3.9/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/tmp/dl-miniconda-0yykfkkk/lib/python3.9/site-packages/datalad/tests/utils.py", line 739, in _wrap_with_tempfile
    return t(*(arg + (filename,)), **kw)
  File "/tmp/dl-miniconda-0yykfkkk/lib/python3.9/site-packages/datalad/tests/utils.py", line 739, in _wrap_with_tempfile
    return t(*(arg + (filename,)), **kw)
  File "/tmp/dl-miniconda-0yykfkkk/lib/python3.9/site-packages/datalad/cmdline/tests/test_main.py", line 387, in test_commanderror_jsonmsgs
    in_('use `git-annex export`', cme.exception.stderr)
AssertionError: 'use `git-annex export`' not found in ''
```
has been failing since seems addition of this test

--- a/datalad/cmdline/tests/test_main.py
+++ b/datalad/cmdline/tests/test_main.py
@@ -329,5 +329,5 @@ def test_commanderror_jsonmsgs(src, exp):
         Runner(cwd=ds.path).run(
             ['datalad', 'push', '--to', 'expdir'],
             protocol=StdOutErrCapture)
-    if ds.repo.git_annex_version >= "8.20200309":
+    if ds.repo.git_annex_version >= "8.20201129":
         in_('use `git-annex export`', cme.exception.stderr)

